### ISSUE TITLE
preds = tf.nn.tanh(conv_t3) * 150 + 255./2

### ISSUE THREAD

**Issue by sguuaa** [lengstrom/fast-style-transfer#79]
I think
preds = tf.nn.tanh(conv_t3) * 150 + 255./2
should be
preds = tf.nn.tanh(conv_t3) * 127.5 + 255./2

**Comment by XHUJOY** [comment]
Original author also use 150 as a multiplier. See
https://github.com/jcjohnson/fast-neural-style/blob/1dc98afdb1b3e5c706bf143591e9865c3053de9b/train.lua#L25
https://github.com/jcjohnson/fast-neural-style/blob/bcacd04cc864eb310894fa5f7af957995a700ae4/fast_neural_style/models.lua#L138

**Comment by sguuaa** [comment]
but why 150?

**Comment by XHUJOY** [comment]
maybe tanh function is hard to output a scalar nearby +/-1

**Comment by lengstrom** [comment]
Empirically multiplying by 150 seems to help with disappearing gradients.
